Sources/SimpleCurrencyConverter/CurrencyConverterProtocol.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+
+public protocol CurrencyConverterProtocol {
+    /**
```

Comment:
We usually use `///` instead of `/**` in our SDKs.

Reply:
Got it! Thank you!